test/cql-pytest/test_describe.py

```diff
@@ -1603,16 +1606,16 @@ def test_desc_roles_with_passwords(cql):
         role_with_pass = "alice"
 
         create_stmt_without_pass = f"CREATE ROLE {role_without_pass} WITH LOGIN = false AND SUPERUSER = false;"
-        create_stmt_with_pass = f"CREATE ROLE {role_with_pass} WITH PASSWORD = 'some_funky_password'"
+        create_stmt_with_pass = f"CREATE ROLE {role_with_pass} WITH PASSWORD = '$2a$10$F5pRau9mKg5abP.DsuPQl.8rQpEoNm3OV91mKjb9vdKPUPejIPq/u'"
 
         cql.execute(create_stmt_without_pass)
         cql.execute(create_stmt_with_pass)
 
-        [salted_hash_result] = cql.execute(f"SELECT salted_hash FROM system.roles WHERE role = '{role_with_pass}'")
-        salted_hash = salted_hash_result.salted_hash
-        create_stmt_with_salted_hash = f"CREATE ROLE {role_with_pass} WITH SALTED HASH = '{sanitize_password(salted_hash)}' AND LOGIN = false AND SUPERUSER = false;"
+        [hashed_password_result] = cql.execute(f"SELECT salted_hash FROM system.roles WHERE role = '{role_with_pass}'")
```

Comment:
also looks like a copy-pasto of sort, ecause the name salted_hash_result was fine - it's the result of SELECT salted_hash :-)